sync-version/action.yaml

```
@@ -1,5 +1,5 @@
-name: Setup ghvmctl
-description: Configure a runner for access to the KVM socket and install ghvmctl
+name: Sync Version
+description: Sync the version with latest upstream releases
```

Thanks for catching this! My bad on the copy/pasta!

```suggestion
description: Sync versions of a snap's applications and parts with the latest upstream versions.
```